tox.ini

```diff
@@ -99,8 +100,8 @@ extras =
     gallery: docs-gallery
 commands =
     pip freeze --all --no-input
-    gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
-    !gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees -D plot_gallery=0 . _build/html {posargs}
+    gallery: {env:SPHINX_COMMAND} {posargs}
+    !gallery: {env:SPHINX_COMMAND} -D plot_gallery=0 {posargs}
```

Comment:
Need to check this will work before a merge since I add the flag at the end of the command.